MdeModulePkg/Test/Mock/Include/GoogleTest/Library/MockCapsuleLib.h

```diff
@@ -0,0 +1,70 @@
+/** @file
+  Google Test mocks for CapsuleLib
+
+  Copyright (c) 2022, Intel Corporation. All rights reserved.

```

Comment:
Are these copyrights correct?

Reply:
Please add our copyright
https://github.com/microsoft/mu_basecore/pull/1059#discussion_r1689018878

Reply:
Copyright Added.

Reply:
Not seeing the Microsoft copyright

Reply:
Added at line 5 in commit : afb608be7ec6b02bccf0103224ea0f3f2ae78e58